From 8f785844bc4e3a7aeec26c6747fd0e12b67b9850 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Sun, 10 Jul 2022 15:38:25 +0100
Subject: [PATCH 1/1] rc: Don't explicity pass compiler to cmake

Rely on CC, CXX env variables instead. Can bork itself with gcc -m32 for example.
---
 ypkg2/rc.yml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/ypkg2/rc.yml b/ypkg2/rc.yml
index 6169179..dafe6f6 100644
--- a/ypkg2/rc.yml
+++ b/ypkg2/rc.yml
@@ -14,13 +14,11 @@ actions:
     - cmake: |
         cmake -DCMAKE_C_FLAGS="${CFLAGS}" -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
         -DCMAKE_LD_FLAGS="${LDFLAGS}" -DCMAKE_LIB_SUFFIX="%LIBSUFFIX%" \
-        -DCMAKE_C_COMPILER="${CC}" -DCMAKE_CXX_COMPILER="${CXX}" \
         -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=%PREFIX%
     - cmake_ninja: |
         cmake -G Ninja . -B solusBuildDir \
         -DCMAKE_C_FLAGS="${CFLAGS}" -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
         -DCMAKE_LD_FLAGS="${LDFLAGS}" -DCMAKE_LIB_SUFFIX="%LIBSUFFIX%" \
-        -DCMAKE_C_COMPILER="${CC}" -DCMAKE_CXX_COMPILER="${CXX}" \
         -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=%PREFIX%
     - meson_configure: |
         CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}" LDFLAGS="${LDFLAGS}" meson --prefix %PREFIX% --buildtype=plain --libdir="lib%LIBSUFFIX%" --libexecdir="lib%LIBSUFFIX%/%PKGNAME%" --sysconfdir=/etc --localstatedir=/var solusBuildDir
-- 
2.35.3

